### PR TITLE
Update throttling dependency

### PR DESCRIPTION
**Pull request by solid-daniilmarchenko**

### CODE REVIEW

File: lib/implementations/debounce_lang_tool_service.dart [hunk 1]
```diff
@@ -20,15 +20,15 @@ class DebounceLangToolService extends LanguageCheckService {
   @override
   Future<Result<List<Mistake>>> findMistakes(String text) async {
     final value =
-        await debouncing.debounce(() => baseService.findMistakes(text))
+        (await debouncing.debounce(() => baseService.findMistakes(text)) ?? [])
             as Result<List<Mistake>>;
```
Comment (yurii-prykhodko-solid) [inline lib/implementations/debounce_lang_tool_service.dart:24]: Casts should be obsolete now.
```suggestion
;
```

File: lib/implementations/throttling_lang_tool_service.dart [hunk 2]
```diff
@@ -30,7 +30,7 @@ class ThrottlingLangToolService extends LanguageCheckService {
 
   @override
   Future<void> dispose() async {
-    await throttling.close();
+    throttling.close();
```
Comment (solid-yuriiprykhodko) [inline lib/implementations/throttling_lang_tool_service.dart:33]: Throttling has a generic arg too, let's remove the typecast

**Review by solid-yuriiprykhodko — CHANGES_REQUESTED** [review]

File: lib/implementations/debounce_lang_tool_service.dart (hunk 1 above)
Comment (solid-daniilmarchenko) [reply lib/implementations/debounce_lang_tool_service.dart:24]: seems like they don't

image

**Review by solid-yuriiprykhodko — APPROVED** [review]
LGTM

**Pull request merged**